common/pom.xml

```diff
@@ -86,6 +86,11 @@
       <artifactId>jackson-databind</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Was guava a transitive dependency before? Should guava be added as a dependency to the drill root pom?